src/sql/src/plan/statement/ddl.rs

```diff
@@ -593,6 +593,13 @@ pub fn plan_create_source(
                 cached_files: None,
             });
             let encoding = get_encoding(format)?;
+
+            if consistency != Consistency::RealTime
```

Comment:
could someone from the sql team confirm if this kind of check is ok / if there is a better way to get what im going for here?

Reply:
I haven't touched the real time consistency code path at all, but given the prevalence of nearly identical checks in this function, this looks like the right approach. If there's a deeper apprehension, lmk and I can do a deeper dive.

Reply:
thanks sean!